src/android/webview.kt

```diff
@@ -36,6 +37,12 @@ open class WebView (context: android.content.Context) : android.webkit.WebView(c
 open class WebViewClient (activity: WebViewActivity) : android.webkit.WebViewClient() {
   protected val activity = WeakReference(activity)
   open protected val TAG = "WebViewClient"
+  open protected var rootDirectory = ""
+
+  fun putRootDirectory(rootDirectory: String) {
+    this.rootDirectory = rootDirectory
```

Comment:
Can you make this `fun Detroit directory (...) ` instead?

Reply:
auto correct typo, please clarify :)

Reply:
oops i meant 
> Can you make this fun setRootDirectory (...) instead?